lightning/kv/importer.go

```diff
@@ -203,6 +203,11 @@ func (importer *Importer) OpenEngine(
 	}, nil
 }
 
+// Tag gets an identification stirng of this engine for logging.
+func (engine *OpenedEngine) Tag() string {
```

Comment:
Seems this method was never used, is it keep consistent with `ClosedEngine.Tag()`?